src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -588,6 +588,10 @@
   <data name="CommandPalette_MoreOptions.[using:Windows.UI.Xaml.Automation]AutomationProperties.HelpText" xml:space="preserve">
     <value>More options</value>
   </data>
+  <data name="CmdEmbeddingDesc" xml:space="preserve">

```

Comment:
This doesn't seem to be used

Reply:
Yeah. I guess since I ended up skipping the normal arg parser mechanics... I don't think I really need to give it the help string. Removing.